internal/pkg/skuba/node/common.go

```diff
@@ -95,4 +87,4 @@ func documentMapToJoinConfiguration(gvkmap kubeadmapi.DocumentMap, allowDeprecat
 	}
 
 	return internalcfg, nil
-}
+}
```

Comment:
I have no idea how this happened. one second i'll fix it because github dislikes it.